dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/tab/DatasetFilesTab.java

```diff
@@ -8,6 +8,7 @@
 import edu.harvard.iq.dataverse.EjbDataverseEngine;
 import edu.harvard.iq.dataverse.PermissionServiceBean;
 import edu.harvard.iq.dataverse.PermissionsWrapper;
+import edu.harvard.iq.dataverse.api.EmbargoAccessService;
```

Comment:
If we want to use `EmbargoAccessService` it shouldn't be in `api` package and it shouldn't extend from `AbstractApiBean`